onboarding/src/Components/hooks/PackagePage.js

```diff
@@ -62,13 +62,12 @@ export function savePackageDetails(handleSuccess, packageId, title, description)
 
 	fetchProps.body = JSON.stringify(data);
 
-	fetch(url + "api/package/" + packageId + "/", fetchProps).then(res => tryFetchJson(res) ).then(
+	fetch(url + "api/packae/" + packageId + "/", fetchProps).then(res => tryFetchJson(res, "Nie udało się zapisać zmian!") ).then(
```

Comment:
Spell error (it seems)
`packae` instead of `package`.
I didn't find any change of API endpoint.

Reply:
I forgot to change it after tests.

Reply:
fixed